### PR TITLE
Allow @frozen and @_fixed_layout for package access level.

### PR DESCRIPTION
**Pull request by elsh**
Package decls were made to be resilient at the definition site by default. By allowing the attributes to be applied to package decls, we can enable non-resilient access.

Resolves rdar://125169361

### CODE REVIEW

**Comment by elsh** [comment]
@swift-ci smoke test

**Review by nkcsgexi — APPROVED** [review]
This makes sense since we have resilient indirections by default now for package decls.

**Comment by elsh** [comment]
@swift-ci smoke test

File: include/swift/AST/DiagnosticsSema.def [hunk 1]
```diff
@@ -6897,8 +6897,8 @@ ERROR(inlinable_dynamic_not_supported,
       none, "'@inlinable' attribute cannot be applied to 'dynamic' declarations", ())
 
 ERROR(inlinable_decl_not_public,
-      none, "'@inlinable' attribute can only be applied to public declarations, "
-      "but %0 is %select{private|fileprivate|internal|package|%error|%error}1",
+      none, "'@inlinable' attribute can only be applied to internal, package, or public declarations, "
+      "but %0 is %select{private|fileprivate|%error|%error|%error|%error}1",
```
Comment (elsh) [inline include/swift/AST/DiagnosticsSema.def:6901]: The definition message here has been outdated; there's NFC wrt this; see TypeCheckAttr.cpp L3230.

**Pull request merged**